### PR TITLE
Merge Github CI changes into master branch

### PR DESCRIPTION
**Pull request by ParkerJones567**
Contains updated CI workflow and bug fixes for updated version of TFLM

### CODE REVIEW

File: CMake/toolchain_LLVM.cmake [hunk 1]
```diff
@@ -26,8 +26,12 @@ set(CMAKE_C_COMPILER clang-14)
 set(CMAKE_CXX_COMPILER clang-14)
 set(CMAKE_ASM_COMPILER clang-14)
 # set(CMAKE_C_LINKER lld-14) # TODO(fabianpedd): doesnt work, need to use -fuse-ld=lld-14 instead
-set(CMAKE_OBJCOPY llvm-objcopy-14)
+
+set(CMAKE_OBJCOPY llvm-objcopy-14)  
+#set(CMAKE_OBJCOPY llvm-objcopy)
```
Comment (PhilippvK) [inline CMake/toolchain_LLVM.cmake:31]: please removed commented lines

File: Integration/MLonMCU/setup_mlonmcu.sh [hunk 2]
```diff
@@ -198,7 +154,11 @@ fi
 # Setup Python #
 ################
 
+<<<<<<< HEAD
+virtualenv -p python3 $VENV_DIR
+=======
```
Comment (PhilippvK) [inline Integration/MLonMCU/setup_mlonmcu.sh:159]: Ooops. Please clean this up

**Review by PhilippvK — CHANGES_REQUESTED** [review]
I saw some "linting" inconsistencies. I normally use the following tools, feel free to try them out:

Python: `black`
CMake: `cmake-format`
Bash: `shellcheck`
C/C++: `clang-format`

We can still merge this IMO. Let's takle automated linting via GitHub Action sometime in the future.

File: CMake/toolchain_LLVM.cmake (hunk 1 above)
Comment (ParkerJones567) [reply CMake/toolchain_LLVM.cmake]: Fixed

File: Integration/MLonMCU/setup_mlonmcu.sh (hunk 2 above)
Comment (ParkerJones567) [reply Integration/MLonMCU/setup_mlonmcu.sh]: Fixed

**Pull request merged**